MiniSim/Service/DeviceService.swift

```diff
@@ -215,7 +216,50 @@ class DeviceService: DeviceServiceProtocol {
 
 // MARK: iOS Methods
 extension DeviceService {
-  private static func parseIOSDevices(result: [String]) -> [Device] {
+  private static func parseIOSPhysicalDevices(result: [String]) -> [Device] {
+    var devices = [Device]()
+
+    let deviceNameIdx = 1
+    let osVersionIdx = 2
+    let identifierIdx = 3
+    var isDevice = false
+    var isOnline = true
```

Comment:
Should we default to the device being online?

Reply:
I personally think it doesnt matter if I set it to true or false. I would rather not set a value but since `isOnline` is set only in the `if let`:
https://github.com/okwasniewski/MiniSim/blob/f60b1bd7991919147ac8ad9102575b01734f5844/MiniSim/Service/DeviceService.swift#L228-L232

then swift will complain that it may not be set and I would not want to make it optional, thats why I add an initial value

The reason I think it doesnt matter is because the value will be set again according to:
https://github.com/okwasniewski/MiniSim/blob/f60b1bd7991919147ac8ad9102575b01734f5844/MiniSim/Service/DeviceService.swift#L230

But I am open to another approach

Reply:
I dont think it matters since we reset the value depending on whether `offline` is found or not.

I just didnt want to make it an optional and leaving it uninitialized makes xcode unhappy because not all paths set it